pkg/config/agent/config.go

```diff
@@ -197,6 +197,10 @@ type AgentConfig struct {
 	// NodeType is type of the Node where Antrea Agent is running.
 	// Defaults to "k8sNode". Valid values include "k8sNode", and "externalNode".
 	NodeType string `yaml:"nodeType,omitempty"`
+	// The namespace to create the ExternalNode for the VM/BM object.
+	// The default value is "default".
+	// It is used only when NodeType is externalNode.
+	Namespace string `yaml:"namespace,omitempty"`
```

Comment:
Add the configuration item into path `build/yamls/externalnode/conf/antrea-agent.conf`